cmd/controller-manager/app/controller_manager.go

```diff
@@ -293,58 +293,76 @@ func StartControllers(s *options.ControllerManagerServer,
 	recorder record.EventRecorder,
 	stop <-chan struct{}) error {
 
-	// Get available service-catalog resources
-	glog.V(5).Info("Getting available resources")
-	availableResources, err := getAvailableResources(serviceCatalogClientBuilder)
+	// When Catalog Controller and Catalog API Server are started at the
+	// same time with API Aggregation enabled, it may take some time before
+	// Catalog registration shows up in API Server.  Attempt to get resources
+	// every 10 seconds and quit after 3 minutes if unsuccessful.
+
+	backoff := wait.Backoff{
+		Duration: 10 * time.Second,
+		Factor:   1,
+		Jitter:   0,
+		Steps:    19,
```

Comment:
Am I misunderstanding or is 19 iterations actually 3 min 10 sec? If so, 18 may be better if for no reason other than to match the above comment.

Reply:
The first poll happens immediately, 18 more gives us a maximum  of 3 minutes polling.